build_projects/dotnet-host-build/StubPackageBuilder.cs

```diff
@@ -82,7 +82,7 @@ private void BuildStubBits(DotNetCli dotnet, string intermediateDirectory)
             projectJson.Append("{");
             projectJson.Append($"  \"version\": \"{version}\",");
             projectJson.Append($"  \"name\": \"{packageId}\",");
-            projectJson.Append("  \"dependencies\": { \"NETStandard.Library\": \"1.5.0-rc2-24008\" },");
+            projectJson.Append("  \"dependencies\": { \"NETStandard.Library\": \"1.5.0-rc2-24027\" },");
```

Comment:
Yes, we will do that in a subsequent change.  This is needed to unblock the build, so adding the quick and dirty fix now.